go.mod

```diff
@@ -1,67 +1,67 @@
 module github.com/digitalocean/csi-digitalocean
 
-go 1.23.5
+go 1.24.1
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/container-storage-interface/spec v1.8.0
-	github.com/digitalocean/go-metadata v0.0.0-20241007135954-c6c417d75f63
-	github.com/digitalocean/godo v1.134.0
-	github.com/docker/docker v26.1.5+incompatible
+	github.com/digitalocean/go-metadata v0.0.0-20250129100319-e3650a3df44b
+	github.com/digitalocean/godo v1.139.0
+	github.com/docker/docker v28.1.1+incompatible
```

Comment:
Looks like this one will have to be downgraded too:
"Error response from daemon: client version 1.49 is too new. Maximum supported API version is 1.48" from e2e failures indicates docker on the runners isn't compatible with 1.28 client